client/components/domains/use-my-domain/utilities/transfer-domain-action.ts

```diff
@@ -78,9 +78,10 @@ export const transferDomainAction: AuthCodeValidationHandler = (
 
 		const startInboundTransferAndReload = async () => {
 			try {
-				const result = await wpcom
-					.undocumented()
-					.startInboundTransfer( selectedSite.ID, domain, authCode );
+				const result = await wpcom.req.get(
+					`/domains/${ encodeURIComponent( domain ) }/inbound-transfer-start/${ selectedSite.ID }`,
+					authCode ? { auth_code: authCode } : {}
```

Comment:
Because of this logic, is it worth extracting this request method to a helper in `lib/domains` for example? Or do you feel it's overkill?

Reply:
Turns out that there already _is_ a `startInboundTransfer` helper in `lib/domains`! So, in 526df0f I used it.